server-common/src/main/java/org/apache/gravitino/server/authentication/OAuthConfig.java

```diff
@@ -73,4 +73,40 @@ public interface OAuthConfig {
           .stringConf()
           .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
           .create();
+
+  // OAuth provider configs
+  ConfigEntry<String> PROVIDER =
+      new ConfigBuilder(OAUTH_CONFIG_PREFIX + "provider")
+          .doc("The OAuth provider to use (e.g., azure)")
+          .version(ConfigConstants.VERSION_1_0_0)
+          .stringConf()
+          .create();
+
+  ConfigEntry<String> CLIENT_ID =
+      new ConfigBuilder(OAUTH_CONFIG_PREFIX + "client-id")
+          .doc("OAuth client ID used for Web UI authentication")
+          .version(ConfigConstants.VERSION_1_0_0)
+          .stringConf()
+          .create();
+
+  ConfigEntry<String> AUTHORITY =
+      new ConfigBuilder(OAUTH_CONFIG_PREFIX + "authority")
+          .doc("OAuth authority URL (authorization server)")
+          .version(ConfigConstants.VERSION_1_0_0)
+          .stringConf()
+          .create();
+
+  ConfigEntry<String> SCOPE =
+      new ConfigBuilder(OAUTH_CONFIG_PREFIX + "scope")
+          .doc("OAuth scopes (space-separated)")
+          .version(ConfigConstants.VERSION_1_0_0)
+          .stringConf()
+          .create();
+
+  ConfigEntry<String> JWKS_URI =
+      new ConfigBuilder(OAUTH_CONFIG_PREFIX + "jwks-uri")
```

Comment:
Could you use camel style name? Gravitino uses camel style although Iceberg may use `-` style name .